### DISCUSSION TITLE
群成员减少或增加

### DISCUSSION THREAD

**Discussion by chengzaizai00** [ttttupup/wxhelper#385] [Ideas]
可以增加一个当有群成员退群或新人入群的消息回调吗

**Comment by ghost** [comment]
这个本身就是有的，监听一下10000的消息就可以了。全在里面。

**Reply by chengzaizai00** [reply]
1000只是群成员增加 或者好友添加    但是当有群成员退群的时候没有回调消息

**Reply by chengzaizai00** [reply]
10000  是一万不好意思

**Reply by ghost** [reply]
退群了 微信上没有提示
不是有读取群成员列表嘛 自己缓存 比对一下

**Reply by chengzaizai00** [reply]
频繁通过接口读取群成员信息  可能会增加微
信被封的风险   其他一些框架是可以实现群成员增加和退出的回调事件
![Uploading QQ截图20240404223630.png…]()
群成员退群  微信服务器肯定是要通知微信客户端更新群成员数量

**Reply by ghost** [reply]
频率可以自己控制 退群不是什么重要事件 实时性要求并不高

非要实时可以看看数据库中是否有群成员数据 利用数据库接口读取

也用过几个其他的框架 并没有看到过你说的这个功能啊
